src/redis-cli.c

```diff
@@ -7699,16 +7699,20 @@ static ssize_t readConn(redisContext *c, char *buf, size_t len)
  * send_sync if 1 means we will explicitly send SYNC command. If 0 means
  * we will not send SYNC command, will send the command that in c->obuf.
  *
- * returns 0 in case an EOF marker is used.
- * Or returns 0 in case a PSYNC +CONTINUE is used. */
-unsigned long long sendSync(redisContext *c, char *out_eof, int send_sync) {
+ * Returns the size of the RDB payload to read, or 0 in case an EOF marker is used and the size
+ * is unknown, also returns 0 in case a PSYNC +CONTINUE was found (no RDB payload).
+ *
+ * The out_full_mode parameter if 1 means this is a full sync, if 0 means this is partial mode. */
+unsigned long long sendSync(redisContext *c, char *out_eof, int *out_full_mode, int send_sync) {
```

Comment:
let's put the output vars last (move the `send_sync` to second)

Reply:
you moved it backwards, but there's still an output variable before it (needs to move further backwards)